gapic-generator-java/src/test/resources/configured_snippet_config.json

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
This PR has 3 similar files.

- test/integration/apis/speech/v1/speech_createCustomClass.json
- test/integration/apis/speech/v1/speech_createCustomClass2.json
- gapic-generator-java/src/test/resources/configured_snippet_config.json

Why?

Reply:
The first file is the one that's actually currently consumed and used by the integration test for `speech`. 

The second file I added as a placeholder to test the ability to intake multiple configuration files (that ability has not yet been added). 

The third file is used for the golden unit test for `ConfiguredSampleComposerTest`. Right now we only have one golden file so its the same file for both the unit test and the integration test; Amanda is working on generating additional golden unit tests that will be added into the `src/test/resources/` directory.

Reply:
TODO: ensure ability to parse multiple configurations is documented somewhere

Reply:
Confirmed this TODO is documented both: https://github.com/googleapis/gapic-generator-java/pull/1190/files#diff-648bf4fdb398cabe3694d2e383a4b273f690c2279af3247ede8c4d0e584a624cR144 and https://github.com/googleapis/gapic-generator-java/pull/1190/files#diff-edf15c9ef2102ff1a52affd916298049af96706c3905003c8332947411fe1811R83